postgres/setup.sh

```diff
@@ -34,26 +34,78 @@ if [ -n "$REPLICATION_USER" ]; then
 
 fi
 
+echo "Applying schema for $POSTGRES_DB"
+psql -v ON_ERROR_STOP=1 -U $WUBLOADER_USER -d $POSTGRES_DB <<-EOSQL
+-- Create type if it doesn't already exist
+DO \$\$ BEGIN
```

Comment:
since you're removing the "IF NOT EXISTS" from everything else, this DO...END block can also go away and you can just do the CREATE TYPE directly.